science/apbs/Portfile

```diff
@@ -3,8 +3,6 @@
 PortSystem              1.0
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               cmake 1.0
-PortGroup               mpi 1.0
-PortGroup               github 1.0
 
 cmake.out_of_source     yes
```

Comment:
Note sure exactly which line you are referring to here... 

PortSystem              1.0
PortGroup               compiler_blacklist_versions 1.0
make.out_of_source     yes

Reply:
well the comment is shown for kine 7, so that's the one I am referring to 😉. Once you're using the latest "cmake" PortGrout the "out_of_source" thing is the default